src/ol/layer/vectortile.js

```diff
@@ -35,17 +35,20 @@ ol.layer.VectorTile = function(opt_options) {
   this.setUseInterimTilesOnError(options.useInterimTilesOnError ?
       options.useInterimTilesOnError : true);
 
-  ol.asserts.assert(options.renderMode == undefined ||
-      options.renderMode == ol.layer.VectorTileRenderType.IMAGE ||
-      options.renderMode == ol.layer.VectorTileRenderType.HYBRID ||
-      options.renderMode == ol.layer.VectorTileRenderType.VECTOR,
-      28); // `renderMode` must be `'image'`, `'hybrid'` or `'vector'`
+  /**
+   * @private
+   * @type {ol.layer.VectorTileRenderType}
+   */
+  this.renderMode_ = ol.layer.VectorTileRenderType.HYBRID;
+  if (options.renderMode) {
+    this.setRenderMode(options.renderMode);
+  }
 
   /**
    * @private
-   * @type {ol.layer.VectorTileRenderType|string}
+   * @type {number}
    */
-  this.renderMode_ = options.renderMode || ol.layer.VectorTileRenderType.HYBRID;
+  this.zDirection_ = options.zDirection || 0;
```

Comment:
I guess this won't happen, but if we ever change the default not to be 0, this would have issues right, i.e. not being able to set 0? Would: ```this.zDirection_ = options.zDirection !== undefined ? options.zDirection : 0;``` be safer here?